README.md

```diff
@@ -64,3 +64,42 @@ To destroy what we just created, you can run the following command.
 ```
 $ terraform destroy
 ```
+`Note:- To make any changes in the created cluster you will need the terraform state files. So don't delete state files of Terraform.`
+
+## Test 
+
+### Configurations
+
+#### Prerequisites
+
+- [Terraform **(~> 0.12.5)**](https://www.terraform.io/downloads.html)
+- [Golang **(~> 1.12.10)**](https://golang.org/dl/)
```

Comment:
We shouldn't need golang to run the terraform script right? It is only needed for the test, so let's remove it from the prereq section.

Reply:
> We shouldn't need golang to run the terraform script right?

Yeah, you're right.
But this prerequisites sub-section is under `## Test` section. Here we mentioned `Golang` for first-time users which haven't the idea about Terratest testing and want to test locally.